### PR TITLE
fixed point can be solved algebraically

### PR DESCRIPTION
**Pull request by Eh2406**
line 341 is:
    parkingstalls = building_bulk * k (a constant for the loop)
and line 351 is:
    building_bulk = orig_bulk - parkingstalls * u (another constant for the loop)
algebra:
    building_bulk = orig_bulk - building_bulk * k * u
    building_bulk + building_bulk * k * u = orig_bulk
    building_bulk * (1 + k * u) = orig_bulk
    building_bulk = orig_bulk / (1 + k * u) 
and the loop is gone

### CODE REVIEW

File: urbansim/developer/sqftproforma.py [hunk 1]
```diff
@@ -336,20 +336,9 @@ def _generate_lookup(self):
...
-                        if np.where(
-                                np.absolute(
-                                    orig_bulk - building_bulk -
-                                    parkingstalls *
-                                    c.parking_sqft_d[parking_config]) > 10.0
-                                )[0].size == 0:
-                            break
-                        building_bulk = orig_bulk - parkingstalls * \
-                            c.parking_sqft_d[parking_config]
+                    building_bulk /= (1.0 + np.sum(uses_distrib * c.parking_rates) *
```
Comment (jiffyclub) [inline urbansim/developer/sqftproforma.py]: pep8 wants the continuation lines lined up with this opening parenthesis:

```python
building_bulk /= (1.0 + np.sum(uses_distrib * c.parking_rates) *
                  c.parking_sqft_d[parking_config] /
                  c.sqft_per_rate)
```

Comment (Eh2406) [inline urbansim/developer/sqftproforma.py]: Thanks! Is that better?

**Comment by jiffyclub** [comment]
Looks good to me, @fscottfoti I'll let you make the call here. Looks like there are some unrelated tests we need to fix.

**Comment by fscottfoti** [comment]
The change makes sense to me - I always wondered if there was an algebraic solution.  @Eh2406 did you check to see if the code gives the same answers?

And yeah, it looks like there are test failures in the transition models but I don't know if any changes have been made there of if it's a Pandas change causing this...

**Comment by jiffyclub** [comment]
My suspicion is a Pandas change.

**Comment by Eh2406** [comment]
@fscottfoti yes, I checked. They matched to within 10. On my cpu the same test pass before and after this commit. I was mostly worried that I had missed some nonlinear step that required a fixed point method. Good thing I did to, I had typed "-" instead of "+" originally.

**Pull request merged**